### DISCUSSION TITLE
EGS-Ray unit integral energy dose

### DISCUSSION THREAD

**Discussion by Stuttga** [nrc-cnrc/EGSnrc#739] [Questions] [answered]
Hello all,

I have a problem with the interpretation of the results of EGS-Ray. Unfortunately there is almost no information about this program, which is based on EGSnrc.

However, I hope that there are parallels to EGSnrc and that someone can tell me in which unit probably the 'Integral absorbed dose' as well as the 'Deposited energy' are given. Generally all input energies are given in MeV. But also those of the results? Or are the results already related to the volume of the parts? Are the Histories included?

I would be very grateful for your help.

**Comment by rtownson** [comment, marked as answer]
In EGSnrc all units of energy are in MeV, and distance in cm unless otherwise specified. Integral absorbed dose will likely be "Gy per initial particle", and deposited energy "MeV per initial particle".

I looked up EGS-Ray briefly and from the one paper I found it looks like a visualization tool based on EGS4/Presta. Note that if it's still using EGS4, that the electron transport is not as accurate as EGSnrc - it is essentially a different MC code and I'm not sure how it has been updated since EGSnrc branched off with new mechanics 20 years ago.

**Comment by rtownson** [comment]
I'll close this issue, but feel free to follow up in the Discussions section or on [reddit](https://www.reddit.com/r/EGSnrc/).

**Comment by Stuttga** [comment]
Thank you so much for your response. You really helped me a lot.

Can you maybe explain to me why you would say that the Ingetral absorbed dose should be "Gy per initial particle" and not also "MeV per initial particle"? I understand the assumption (because it is obviously a dose which is in general given in Gy) - but unfortunately this makes absolutely no sense from the numerical results, because the doses would be incredibly high. Therefore (with respect to the results of the simulation) it would make more sense if the unit of the Integral absorbed dose would be the energy in "MeV per initial particle". You think that this is also possible or rather unlikely?

I'm almost going crazy with this program because i can't interpret the results of the simulation for sure. Even if it is a great program to visualize different situations in which radiation is transmitted.

**Comment by ftessier** [comment]
@Stuttga Reporting the dose is essentially "equivalent" to reporting the deposited energy (MeV). Internally, EGSnrc (and maybe EGS-Ray?) computes the energy deposited in geometrical regions, and divides by the region mass just for the result output. Perhaps the mass of the region is set incorrectly? At any rate I do not know what EGS-Ray does, or if it handles the normalization correctly, but as Reid mentioned is is based on the legacy version EGS4, which suffers from significant electron step-size artifacts.

Hence I encourage you to use EGSnrc and `egs_view` for your computation and visualization, either as your main tool, or at least as a check for the EGS-Ray numbers.

**Comment by rtownson** [comment]
@Stuttga it sounds like you might be trying to convert "Gy per initial particle" into absolute dose, in Gy. This is not a trivial thing to do, since in reality it's impossible to know the number of initial particles. The only way I would suggest doing this is by performing a calibration simulation where you determine a calibration factor between the simulation and experimental results under stable conditions - even this has limited success and applicability. In general, MC simulations are most powerful when used for relative comparisons.
